flixel/input/mouse/FlxMouse.hx

```diff
@@ -192,11 +197,6 @@ class FlxMouse extends FlxPointer implements IFlxInputManager
 	var _rightButton:FlxMouseButton;
 	#end
 
-	/**
-	 * This is just a reference to the current cursor image, if there is one.
-	 */
-	var _cursor:Bitmap = null;
```

Comment:
we need to keep this for backwards compatibility, in case anyone is using `@:privateAccess`
```hx
var _cursor(get, set):Bitmap = null;

inline function get__cursor()
{
    return cursor;
}

inline function set__cursor(value:Bitmap)
{
    return cursor = value;
}
```

Reply:
sorry,  need to remove the ` = null`